share_app/src/main/java/org/odk/share/controller/Wifi.java

```diff
@@ -0,0 +1,29 @@
+package org.odk.share.controller;
+
+import android.content.Context;
+import android.net.wifi.ScanResult;
+import android.net.wifi.WifiManager;
+
+/**
+ * Created by laksh on 5/22/2018.
+ */
+
+public class Wifi {
```

Comment:
The name of the class does not reflect what it is doing. A better name could be `WifiHelper` or something else